scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json

```diff
@@ -0,0 +1,278 @@
+{
+  "tables": [
+    {
+      "name": "activity",
+      "columns": [
+        {"name": "activity_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "activity_type", "datatype": "string", "array_of": false},
+        {"name": "used_file_id", "datatype": "string", "array_of": true},
+        {"name": "generated_file_id", "datatype": "string", "array_of": true}
+      ],
+      "primaryKey": ["activity_id"],
+      "partitionMode": "none",
```

Comment:
What are `partitionMode`, `datePartitionOptions` and `intPartitionOptions` used for?
Are these hints to the Azul indexer to aid efficient parallel indexing, or something else?

Reply:
These are used by TDR as instructions on how to partition the data in BigQuery.